tests/unit/cli/test_utils.py

```diff
@@ -153,6 +153,33 @@ def test_content_gets_decoded(self, content):
         assert result == 'FROM centos7'
 
 
+class TestFormatMetadataQuery:
```

Comment:
there are no tests for `metadata_type` or `imageDigest`

Reply:
added